packages/gatsby-theme-newrelic/src/components/Skeleton.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { css } from '@emotion/react';
+
+const Skeleton = () => (
+  <div
+    css={css`
+      width: 100px;
+      height: 100px;
+      animation: skeleton-loading 1s linear infinite alternate;
+      @keyframes skeleton-loading {
```

Comment:
this is looking great, one thing that still remains is to account for dark mode. We'll want to add styles for when the dark mode class name is applied to the page. It should look something like this: 
https://github.com/newrelic/gatsby-theme-newrelic/blob/e92c4259148b0349968629029f5a3cbaa41dce7f/packages/gatsby-theme-newrelic/src/components/TextHighlight.js#L25-L27